helm/cluster/ci/ci-values.yaml

```diff
@@ -86,6 +86,15 @@ internal:
           - PodSecurityPolicy
           apiAudiences:
             templateName: "cluster.test.kubeadmControlPlane.kubeadmConfigSpec.clusterConfiguration.apiServer.apiAudiences"
+          etcdPrefix: giantswarm.io
```

Comment:
I'm wondering if it would make sense to move `etcdPrefix` into `etcd` object

Reply:
It is API server CLI flag (related to etcd usage), see here https://v1-24.docs.kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/

```
--etcd-prefix string     Default: "/registry"

The prefix to prepend to all resource paths in etcd.
```

Reply:
All good, just thought that the contract would be nicer having everything in `etcd` 🙂. Thanks 👍🏻